SurfaceTopography/ChangeLog.md

```diff
@@ -1,11 +1,12 @@
 Change log for SurfaceTopography
 =============================
 
-Deprecations in version 1.0
----------------------------
+v1.0 (not yet released)
+-----------------------
 
-- `unit` entry for the `info` dictionary will disappear
-- `scale_factor` property for scaled topographies will disappear
+- ENH: Generalized HDF5 reader
+- MAINT: Deprecated `unit` entry of `info` dictionary
```

Comment:
Is this "deprecated" or "removed"? It looks like that it has been removed here.
From my understanding, "deprecated" means "considered for removal in a future release".

Reply:
You are right, it has been removed. We deprecated this a while ago.